pkg/nfd-master/nfd-master.go

```diff
@@ -78,6 +79,11 @@ type Args struct {
 	ResourceLabels       utils.StringSetVal
 }
 
+type DeniedNs struct {
```

Comment:
nitty nit: we don't need to export this type

```suggestion
type deniedNs struct {
```